Pipeline/controllers/PipelineController.py

```diff
@@ -1,21 +1,72 @@
-# This is where the "business logic" of the pipeline will live for the TwoAdder service
-# basically whatever data manipulation, or combination of service calls we need to do will happen here
-# the routing functions will basically call the controller functions to do what needs to be done
-from services.OneAdder import adder as oneAdder
-from services.TwoAdder import adder as twoAdder
-from services.ThreeAdder import adder as threeAdder
+from .BayesPairingController import BayesPairing
+
 
 class Pipeline:
 
     @staticmethod
-    def additionChain(input):
-        try:
-            OneResult = str(oneAdder(input)["result"])
-            TwoResult = str(twoAdder(OneResult)["result"])
-            ThreeResult = threeAdder(TwoResult) #Do not extract, send as JSON
-            return ThreeResult
-            
-        except:
-            return 'Invalid Input'
-
-        
\ No newline at end of file
+    def pipeline_file(arguments, files):
+        '''
+        Processes data for a call to the pipeline and makes calls to the relevant services
+
+        :param arguments: a dictionary containing all arguments for any service
+        :param files: a dictionary containing any relevant files including BayesPairing input
+        :returns: a response containing all relevant output
+        '''
+        bp_input = files.get("bp_input")
+        rnamigos_library = files.get("rnamigos_library")
+        vernal = arguments.get("vernal", default=1, type=int)
+        rnamigos = arguments.get("rnamigos", default=1, type=int)
```

Comment:
so from my understanding, the reason you're using `.get()` here is because of the request body being formdata?

Reply:
Its an ImmutableMultiDict